### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by maximuk**
# What does this PR do:
* added `theme-builder` package
* added `css-themes-polyfill` package
* init lerna
# Additional info:
* all commits form `theme-builder/master` and `css-themes-polyfill/master` added to this repository

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+	"devDependencies": {
```
Comment (yurist38) [inline package.json]: Tabs?

Comment (maximuk) [reply package.json]: The file was created by `lerna`

Comment (AlleeX) [reply package.json]: Ohh.. @maximuk then you should spank `lerna` =)

File: packages/css-themes-polyfill/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/Travix-International/css-themes-polyfill",
+  "contributors": [
+    {
+      "name": "Artem Riasnianskyi",
+      "url": "https://github.com/asci"
+    }
+  ],
+  "devDependencies": {
```
Comment (fahad19) [inline packages/css-themes-polyfill/package.json]: move `devDependencies` of all packages to root `package.json` file.

File: packages/theme-builder/package.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    {
+      "name": "Ivana Viranovska",
+      "url": "https://github.com/iwwwi"
+    },
+    {
+      "name": "Artem Riasnianskyi",
+      "url": "https://github.com/asci"
+    }
+  ],
+  "devDependencies": {
```
Comment (fahad19) [inline packages/theme-builder/package.json]: move `devDependencies` of all packages to root `package.json` file.

File: lerna.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.6.0",
+  "packages": [
+    "packages/*"
+  ],
+  "version": "independent"
```
Comment (fahad19) [inline lerna.json]: if in a monorepo, better have common version numbers.

File: lerna.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fahad19) [inline lerna.json]: integrate with `lerna-changelog` too.

File: packages/css-themes-polyfill/.eslintrc [hunk 6]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (fahad19) [inline packages/css-themes-polyfill/.eslintrc]: move all dotfiles to root of the repository.

File: packages/css-themes-polyfill/.editorconfig [hunk 7]
```diff
@@ -0,0 +1,15 @@
+; This file is for unifying the coding style for different editors and IDEs.
```
Comment (fahad19) [inline packages/css-themes-polyfill/.editorconfig]: move all dotfiles to root of the repository.

File: packages/css-themes-polyfill/LICENSE [hunk 8]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (fahad19) [inline packages/css-themes-polyfill/LICENSE]: the monorepo can have a single LICENSE file in the root of the repo.

File: packages/css-themes-polyfill/MAINTAINERS.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
+# Maintainers
```
Comment (fahad19) [inline packages/css-themes-polyfill/MAINTAINERS.md]: only one MAINTAINERS.md file needed for a repo. can be removed from package-level.

File: packages/css-themes-polyfill/Makefile [hunk 10]
```diff
@@ -0,0 +1,26 @@
+VERSION := patch
+GITHUB_API_TOKEN := ""
+
+release:
```
Comment (fahad19) [inline packages/css-themes-polyfill/Makefile]: with the introduction of `lerna` and `$ lerna publish`, this `make release` is now redundant.

File: packages/css-themes-polyfill/package.json [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+  "description": "Travix CSS variables for themes polyfill",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "mkdir -p lib && uglifyjs --compress --mangle --output lib/index.js src/index.js",
+    "test": "node test/index.spec.js",
+    "lint": "eslint --color src/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Travix-International/css-themes-polyfill.git"
```
Comment (fahad19) [inline packages/css-themes-polyfill/package.json]: point to this repo instead.

File: packages/theme-builder/.babelrc [hunk 12]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (fahad19) [inline packages/theme-builder/.babelrc]: all dotfiles can move to root of the repo.

File: packages/theme-builder/.travis.yml [hunk 13]
```diff
@@ -0,0 +1,13 @@
+language: node_js
```
Comment (fahad19) [inline packages/theme-builder/.travis.yml]: this file has no impact.

**Comment by fahad19** [comment]
was it done using `$ lerna import`? https://github.com/lerna/lerna#import

**Comment by maximuk** [comment]
@fahad19 I didn't use `lerna import`. I added git remotes and merged them to this repository

File: .babelrc [hunk 14]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
```
Comment (yurist38) [inline .babelrc]: Why not `Travix` preset?

**Comment by froskie** [comment]
Closing pull request due to previous changes in the repo.

**Pull request closed without merging**